yew-ui/index.html

```diff
@@ -2,7 +2,7 @@
 <html class="dark">
     <head>
         <meta charset="utf-8" />
-        <title>videocall.rs</title>
+        <title>Concept Car</title>
```

Comment:
seems like this is from the fork, can we make it a variable so that we can share this code?